src/test/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessorTest.java

```diff
@@ -536,35 +536,34 @@ void GIVEN_unstructured_log_WHEN_breaches_event_size_and_batch_size_limits_THEN_
             assertNotNull(logEventsForStream1.getLogEvents());
             // Total log events: 1 (first line) + 2 (second line) + 2 (third line) = 5
             assertEquals(5, logEventsForStream1.getLogEvents().size());
-            assertTrue(logEventsForStream1.getAttemptLogFileInformationMap().containsKey(file.getAbsolutePath()));
-            assertEquals(0, logEventsForStream1.getAttemptLogFileInformationMap().get(file.getAbsolutePath())
+            assertTrue(logEventsForStream1.getAttemptLogFileInformationMap().containsKey(fileHash));
+            assertEquals(0, logEventsForStream1.getAttemptLogFileInformationMap().get(fileHash)
                     .getStartPosition());
             assertEquals(expectedBytesRead,
-                    logEventsForStream1.getAttemptLogFileInformationMap().get(file.getAbsolutePath()).getBytesRead());
+                    logEventsForStream1.getAttemptLogFileInformationMap().get(fileHash).getBytesRead());
             assertEquals("TestComponent", logEventsForStream1.getComponentName());
 
         } finally {
-            assertTrue(file.delete());
+            assertTrue(logFile.delete());
         }
     }
     @Test
-    void GIVEN_null_log_message_WHEN_upload_attempted_THEN_null_message_considered_as_unstructured_log(ExtensionContext ec) throws IOException {
```

Comment:
this test is now testing the exact opposite of what it was testing before. Why are we changing the behavior? Is the behavior change correct?